packages/component/src/hooks/internal/useAcknowledgedActivity.js

```diff
@@ -0,0 +1,61 @@
+import { hooks } from 'botframework-webchat-api';
+import { useMemo, useRef } from 'react';
+import { useSticky } from 'react-scroll-to-bottom';
+
+import findLastIndex from '../../Utils/findLastIndex';
+import getActivityUniqueId from '../../Utils/getActivityUniqueId';
+import useChanged from './useChanged';
+
+const { useActivities } = hooks;
+
+// Acknowledged means either:
+// 1. The user sent a message
+//    - We don't need a condition here. When Web Chat send the message, it will scroll to bottom, and it will trigger condition 2 below.
+// 2. At the bottom of the transcript, scrolled from a non-bottom scroll position
+//    - If the transcript is already at the bottom, the user need to scroll up and then go back down
+//    - What happen if we are relaxing "scrolled from a non-bottom scroll position":
```

Comment:
```suggestion
//    - What happens if we are relaxing "scrolled from a non-bottom scroll position":
```

Reply:
So are you saying that with this PR, the new behavior is 'relaxed scrolled from a non-bottom scroll position'?
Don't use future tense in that sense - if this is the commit that's making changes, it needs to be phrased in present tense.

This seems more like commentary that should go in change description, not necessarily as code comments (e.g. in PR summary/release summary). What do you think?